### DISCUSSION TITLE
Can we remove Logistic model ? (Poll)

### DISCUSSION THREAD

**Discussion by ravi7522** [ravi7522/Football-Prediction#17] [Polls]
![image](https://github.com/ravi7522/Football-Prediction/assets/23443691/d24d1da9-6ec2-4d96-a987-164f04418990)

This **confusion matrix** is for **Logistic regression** model. Here we can see that we are **not able to predict accurately** for the **Loss**. We can not see any **prediction** which are **true**. Mostly those prediction are **False**

**Comment by zeelsheladiya** [comment]
I guess we can still include this model. so we can compare and I would data still need to do feature extraction. so let it be like this. moreover, anyway we also gonna make DNN

**Reply by ravi7522** [reply]
But as you can see that the prediction of 1 which is away win we are not getting any important true predictions. Thats why i am asking to remove it.

**Comment by babuaditya97** [comment]
will there be any change if we do some tuning? because its always better to have a decent simple explanatory model.
